README.md

```diff
@@ -1,155 +1,236 @@
 # flutter_flux
 
-> A Dart app architecture library with uni-directional data flow inspired by [RefluxJS](https://github.com/reflux/refluxjs) and Facebook's [Flux](https://facebook.github.io/flux/).
+A Dart/[Flutter](https://flutter.io) app architecture library with uni-directional data flow inspired by [RefluxJS](https://github.com/reflux/refluxjs) and Facebook's [Flux](https://facebook.github.io/flux/). [Flux](https://facebook.github.io/flux/) is the complementary “model” framework also designed by Facebook. Flutter_flux is a reimplementation of Flux that uses idiomatic Dart and integrates cleanly with Flutter.
```

Comment:
I'd say "A library for Flutter that implements the Flux app architecture pattern, and helps enable a uni-directional data flow...."

Drop the "Dart" part.